### ISSUE TITLE
Rollup failed to resolve import "#MJX-1-TEX-I-1D449

### ISSUE THREAD

**Issue by kdheepak** [bluwy/svelte-preprocess-import-assets#4]
Thanks for making this plugin!

I'm getting the following error when using rehype-mathjax:

```
[vite]: Rollup failed to resolve import "#MJX-1-TEX-I-1D449" from "filename.md".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
> [vite]: Rollup failed to resolve import "#MJX-1-TEX-I-1D449 from "filename.md"
```

I believe this plugin is trying to resolve a anchor link import? Any idea what exactly might be going on here and how to fix it?

**Comment by kdheepak** [comment]
I was able to solve this by using a filter. I think this plugin should ignore anchor links by default though. Thoughts? I'll close this since this is not a issue but more a feature request, and you can decide if you want to reopen / implement this.

**Comment by bluwy** [comment]
Hi, this plugin does ignore anchor links by default, or maybe you're referring to something else? I think it makes sense to fix this bug in this package itself, can you show how the markup looks like after going through `rehype-mathjax`? So I can create a test case for this.

**Comment by kdheepak** [comment]
This is the markdown that I'm trying to render:

```
$$x + (x - 1) + (x - 2) + (x - 3) + \ldots + 1 >= N$$
```

This is the render code I'm using:

```
import { unified } from 'unified'
import rehypeParse from 'rehype-parse'
import rehypeStringify from 'rehype-stringify'
import rehypeMathjaxSvg from 'rehype-mathjax'

const c  = "$$x + (x - 1) + (x - 2) + (x - 3) + \ldots + 1 >= N$$"
const markdown2svelte = unified()
  .use(rehypeParse, {fragment: true, emitParseErrors: true})
  .use(rehypeMathjaxSvg)
  .use(rehypeStringify, {allowDangerousHtml: false})
const result = await markdown2svelte()
  .process(c)
```

This is the mathjax that gets generated:

```
<p><span class="math display math-display"><mjx-container class="MathJax" jax="SVG" display="true"><svg style="vertical-align: -0.566ex;" xmlns="http://www.w3.org/2000/svg" width="46.546ex" height="2.262ex" role="img" focusable="false" viewBox="0 -750 20573.1 1000" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="MJX-1-TEX-I-1D465" d="M52 289Q59 331 106 386T222 442Q257 442 286 424T329 379Q371 442 430 442Q467 442 494 420T522 361Q522 332 508 314T481 292T458 288Q439 288 427 299T415 328Q415 374 465 391Q454 404 425 404Q412 404 406 402Q368 386 350 336Q290 115 290 78Q290 50 306 38T341 26Q378 26 414 59T463 140Q466 150 469 151T485 153H489Q504 153 504 145Q504 144 502 134Q486 77 440 33T333 -11Q263 -11 227 52Q186 -10 133 -10H127Q78 -10 57 16T35 71Q35 103 54 123T99 143Q142 143 142 101Q142 81 130 66T107 46T94 41L91 40Q91 39 97 36T113 29T132 26Q168 26 194 71Q203 87 217 139T245 247T261 313Q266 340 266 352Q266 380 251 392T217 404Q177 404 142 372T93 290Q91 281 88 280T72 278H58Q52 284 52 289Z"></path><path id="MJX-1-TEX-N-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path><path id="MJX-1-TEX-N-28" d="M94 250Q94 319 104 381T127 488T164 576T202 643T244 695T277 729T302 750H315H319Q333 750 333 741Q333 738 316 720T275 667T226 581T184 443T167 250T184 58T225 -81T274 -167T316 -220T333 -241Q333 -250 318 -250H315H302L274 -226Q180 -141 137 -14T94 250Z"></path><path id="MJX-1-TEX-N-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path><path id="MJX-1-TEX-N-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path><path id="MJX-1-TEX-N-29" d="M60 749L64 750Q69 750 74 750H86L114 726Q208 641 251 514T294 250Q294 182 284 119T261 12T224 -76T186 -143T145 -194T113 -227T90 -246Q87 -249 86 -250H74Q66 -250 63 -250T58 -247T55 -238Q56 -237 66 -225Q221 -64 221 250T66 725Q56 737 55 738Q55 746 60 749Z"></path><path id="MJX-1-TEX-N-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path id="MJX-1-TEX-N-33" d="M127 463Q100 463 85 480T69 524Q69 579 117 622T233 665Q268 665 277 664Q351 652 390 611T430 522Q430 470 396 421T302 350L299 348Q299 347 308 345T337 336T375 315Q457 262 457 175Q457 96 395 37T238 -22Q158 -22 100 21T42 130Q42 158 60 175T105 193Q133 193 151 175T169 130Q169 119 166 110T159 94T148 82T136 74T126 70T118 67L114 66Q165 21 238 21Q293 21 321 74Q338 107 338 175V195Q338 290 274 322Q259 328 213 329L171 330L168 332Q166 335 166 348Q166 366 174 366Q202 366 232 371Q266 376 294 413T322 525V533Q322 590 287 612Q265 626 240 626Q208 626 181 615T143 592T132 580H135Q138 579 143 578T153 573T165 566T175 555T183 540T186 520Q186 498 172 481T127 463Z"></path><path id="MJX-1-TEX-N-2026" d="M78 60Q78 84 95 102T138 120Q162 120 180 104T199 61Q199 36 182 18T139 0T96 17T78 60ZM525 60Q525 84 542 102T585 120Q609 120 627 104T646 61Q646 36 629 18T586 0T543 17T525 60ZM972 60Q972 84 989 102T1032 120Q1056 120 1074 104T1093 61Q1093 36 1076 18T1033 0T990 17T972 60Z"></path><path id="MJX-1-TEX-N-3E" d="M84 520Q84 528 88 533T96 539L99 540Q106 540 253 471T544 334L687 265Q694 260 694 250T687 235Q685 233 395 96L107 -40H101Q83 -38 83 -20Q83 -19 83 -17Q82 -10 98 -1Q117 9 248 71Q326 108 378 132L626 250L378 368Q90 504 86 509Q84 513 84 520Z"></path><path id="MJX-1-TEX-N-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path id="MJX-1-TEX-I-1D441" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D465" xlink:href="#MJX-1-TEX-I-1D465"></use></g><g data-mml-node="mo" transform="translate(794.2,0)"><use data-c="2B" xlink:href="#MJX-1-TEX-N-2B"></use></g><g data-mml-node="mo" transform="translate(1794.4,0)"><use data-c="28" xlink:href="#MJX-1-TEX-N-28"></use></g><g data-mml-node="mi" transform="translate(2183.4,0)"><use data-c="1D465" xlink:href="#MJX-1-TEX-I-1D465"></use></g><g data-mml-node="mo" transform="translate(2977.7,0)"><use data-c="2212" xlink:href="#MJX-1-TEX-N-2212"></use></g><g data-mml-node="mn" transform="translate(3977.9,0)"><use data-c="31" xlink:href="#MJX-1-TEX-N-31"></use></g><g data-mml-node="mo" transform="translate(4477.9,0)"><use data-c="29" xlink:href="#MJX-1-TEX-N-29"></use></g><g data-mml-node="mo" transform="translate(5089.1,0)"><use data-c="2B" xlink:href="#MJX-1-TEX-N-2B"></use></g><g data-mml-node="mo" transform="translate(6089.3,0)"><use data-c="28" xlink:href="#MJX-1-TEX-N-28"></use></g><g data-mml-node="mi" transform="translate(6478.3,0)"><use data-c="1D465" xlink:href="#MJX-1-TEX-I-1D465"></use></g><g data-mml-node="mo" transform="translate(7272.6,0)"><use data-c="2212" xlink:href="#MJX-1-TEX-N-2212"></use></g><g data-mml-node="mn" transform="translate(8272.8,0)"><use data-c="32" xlink:href="#MJX-1-TEX-N-32"></use></g><g data-mml-node="mo" transform="translate(8772.8,0)"><use data-c="29" xlink:href="#MJX-1-TEX-N-29"></use></g><g data-mml-node="mo" transform="translate(9384,0)"><use data-c="2B" xlink:href="#MJX-1-TEX-N-2B"></use></g><g data-mml-node="mo" transform="translate(10384.2,0)"><use data-c="28" xlink:href="#MJX-1-TEX-N-28"></use></g><g data-mml-node="mi" transform="translate(10773.2,0)"><use data-c="1D465" xlink:href="#MJX-1-TEX-I-1D465"></use></g><g data-mml-node="mo" transform="translate(11567.4,0)"><use data-c="2212" xlink:href="#MJX-1-TEX-N-2212"></use></g><g data-mml-node="mn" transform="translate(12567.7,0)"><use data-c="33" xlink:href="#MJX-1-TEX-N-33"></use></g><g data-mml-node="mo" transform="translate(13067.7,0)"><use data-c="29" xlink:href="#MJX-1-TEX-N-29"></use></g><g data-mml-node="mo" transform="translate(13678.9,0)"><use data-c="2B" xlink:href="#MJX-1-TEX-N-2B"></use></g><g data-mml-node="mo" transform="translate(14679.1,0)"><use data-c="2026" xlink:href="#MJX-1-TEX-N-2026"></use></g><g data-mml-node="mo" transform="translate(16073.3,0)"><use data-c="2B" xlink:href="#MJX-1-TEX-N-2B"></use></g><g data-mml-node="mn" transform="translate(17073.6,0)"><use data-c="31" xlink:href="#MJX-1-TEX-N-31"></use></g><g data-mml-node="mo" transform="translate(17851.3,0)"><use data-c="3E" xlink:href="#MJX-1-TEX-N-3E"></use><use data-c="3D" xlink:href="#MJX-1-TEX-N-3D" transform="translate(778,0)"></use></g><g data-mml-node="mi" transform="translate(19685.1,0)"><use data-c="1D441" xlink:href="#MJX-1-TEX-I-1D441"></use></g></g></g></svg></mjx-container></span></p>
```

This is the screenshot of what it is supposed to look like:

<img width="370" alt="Screen Shot 2022-01-13 at 10 07 13 PM" src="https://user-images.githubusercontent.com/1813121/149454949-1647ab73-ee93-4b79-b9ab-7d8e6a3de400.png">

And when I use this plugin without any filter, this is the script tag that gets generated:

```
<script>import ___ASSET___0 from "#MJX-1-TEX-I-1D465";import ___ASSET___1 from "#MJX-1-TEX-N-2B";import ___ASSET___2 from "#MJX-1-TEX-N-28";import ___ASSET___3 from "#MJX-1-TEX-N-2212";import ___ASSET___4 from "#MJX-1-TEX-N-31";import ___ASSET___5 from "#MJX-1-TEX-N-29";import ___ASSET___6 from "#MJX-1-TEX-N-32";import ___ASSET___7 from "#MJX-1-TEX-N-33";import ___ASSET___8 from "#MJX-1-TEX-N-2026";import ___ASSET___9 from "#MJX-1-TEX-N-3E";import ___ASSET___10 from "#MJX-1-TEX-N-3D";import ___ASSET___11 from "#MJX-1-TEX-I-1D441";import ___ASSET___12 from "#MJX-2-TEX-LO-2211";import ___ASSET___13 from "#MJX-2-TEX-I-1D458";import ___ASSET___14 from "#MJX-2-TEX-N-3D";import ___ASSET___15 from "#MJX-2-TEX-N-31";import ___ASSET___16 from "#MJX-2-TEX-I-1D465";import ___ASSET___17 from "#MJX-2-TEX-N-3E";import ___ASSET___18 from "#MJX-2-TEX-I-1D441";import ___ASSET___19 from "#MJX-3-TEX-I-1D465";import ___ASSET___20 from "#MJX-3-TEX-N-D7";import ___ASSET___21 from "#MJX-3-TEX-N-28";import ___ASSET___22 from "#MJX-3-TEX-N-31";import ___ASSET___23 from "#MJX-3-TEX-N-2B";import ___ASSET___24 from "#MJX-3-TEX-N-29";import ___ASSET___25 from "#MJX-3-TEX-N-32";import ___ASSET___26 from "#MJX-3-TEX-N-3E";import ___ASSET___27 from "#MJX-3-TEX-N-3D";import ___ASSET___28 from "#MJX-3-TEX-I-1D441";import ___ASSET___29 from "#MJX-4-TEX-I-1D43F";import ___ASSET___30 from "#MJX-4-TEX-I-1D434";import ___ASSET___31 from "#MJX-4-TEX-I-1D447";import ___ASSET___32 from "#MJX-4-TEX-I-1D438";import ___ASSET___33 from "#MJX-4-TEX-I-1D44B";
```

And this is the markup that gets generated (everything gets inlined and minified and I'm not able to render this to confirm, this is from the print output of a custom svelte preprocessor that I wrote before the error occurs):

```
<p><span class="math display math-display"><mjx-container class="MathJax" jax="SVG" display="true"><svg style="vertical-align: -0.566ex;" xmlns="http://www.w3.org/2000/svg" width="46.546ex" height="2.262ex" role="img" focusable="false" viewBox="0 -750 20573.1 1000" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="MJX-1-TEX-I-1D465" d="M52 289Q59 331 106 386T222 442Q257 442 286 424T329 379Q371 442 430 442Q467 442 494 420T522 361Q522 332 508 314T481 292T458 288Q439 288 427 299T415 328Q415 374 465 391Q454 404 425 404Q412 404 406 402Q368 386 350 336Q290 115 290 78Q290 50 306 38T341 26Q378 26 414 59T463 140Q466 150 469 151T485 153H489Q504 153 504 145Q504 144 502 134Q486 77 440 33T333 -11Q263 -11 227 52Q186 -10 133 -10H127Q78 -10 57 16T35 71Q35 103 54 123T99 143Q142 143 142 101Q142 81 130 66T107 46T94 41L91 40Q91 39 97 36T113 29T132 26Q168 26 194 71Q203 87 217 139T245 247T261 313Q266 340 266 352Q266 380 251 392T217 404Q177 404 142 372T93 290Q91 281 88 280T72 278H58Q52 284 52 289Z"></path><path id="MJX-1-TEX-N-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path><path id="MJX-1-TEX-N-28" d="M94 250Q94 319 104 381T127 488T164 576T202 643T244 695T277 729T302 750H315H319Q333 750 333 741Q333 738 316 720T275 667T226 581T184 443T167 250T184 58T225 -81T274 -167T316 -220T333 -241Q333 -250 318 -250H315H302L274 -226Q180 -141 137 -14T94 250Z"></path><path id="MJX-1-TEX-N-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path><path id="MJX-1-TEX-N-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path><path id="MJX-1-TEX-N-29" d="M60 749L64 750Q69 750 74 750H86L114 726Q208 641 251 514T294 250Q294 182 284 119T261 12T224 -76T186 -143T145 -194T113 -227T90 -246Q87 -249 86 -250H74Q66 -250 63 -250T58 -247T55 -238Q56 -237 66 -225Q221 -64 221 250T66 725Q56 737 55 738Q55 746 60 749Z"></path><path id="MJX-1-TEX-N-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path id="MJX-1-TEX-N-33" d="M127 463Q100 463 85 480T69 524Q69 579 117 622T233 665Q268 665 277 664Q351 652 390 611T430 522Q430 470 396 421T302 350L299 348Q299 347 308 345T337 336T375 315Q457 262 457 175Q457 96 395 37T238 -22Q158 -22 100 21T42 130Q42 158 60 175T105 193Q133 193 151 175T169 130Q169 119 166 110T159 94T148 82T136 74T126 70T118 67L114 66Q165 21 238 21Q293 21 321 74Q338 107 338 175V195Q338 290 274 322Q259 328 213 329L171 330L168 332Q166 335 166 348Q166 366 174 366Q202 366 232 371Q266 376 294 413T322 525V533Q322 590 287 612Q265 626 240 626Q208 626 181 615T143 592T132 580H135Q138 579 143 578T153 573T165 566T175 555T183 540T186 520Q186 498 172 481T127 463Z"></path><path id="MJX-1-TEX-N-2026" d="M78 60Q78 84 95 102T138 120Q162 120 180 104T199 61Q199 36 182 18T139 0T96 17T78 60ZM525 60Q525 84 542 102T585 120Q609 120 627 104T646 61Q646 36 629 18T586 0T543 17T525 60ZM972 60Q972 84 989 102T1032 120Q1056 120 1074 104T1093 61Q1093 36 1076 18T1033 0T990 17T972 60Z"></path><path id="MJX-1-TEX-N-3E" d="M84 520Q84 528 88 533T96 539L99 540Q106 540 253 471T544 334L687 265Q694 260 694 250T687 235Q685 233 395 96L107 -40H101Q83 -38 83 -20Q83 -19 83 -17Q82 -10 98 -1Q117 9 248 71Q326 108 378 132L626 250L378 368Q90 504 86 509Q84 513 84 520Z"></path><path id="MJX-1-TEX-N-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path id="MJX-1-TEX-I-1D441" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D465" xlink:href="{___ASSET___0}"></use></g><g data-mml-node="mo" transform="translate(794.2,0)"><use data-c="2B" xlink:href="{___ASSET___1}"></use></g><g data-mml-node="mo" transform="translate(1794.4,0)"><use data-c="28" xlink:href="{___ASSET___2}"></use></g><g data-mml-node="mi" transform="translate(2183.4,0)"><use data-c="1D465" xlink:href="{___ASSET___0}"></use></g><g data-mml-node="mo" transform="translate(2977.7,0)"><use data-c="2212" xlink:href="{___ASSET___3}"></use></g><g data-mml-node="mn" transform="translate(3977.9,0)"><use data-c="31" xlink:href="{___ASSET___4}"></use></g><g data-mml-node="mo" transform="translate(4477.9,0)"><use data-c="29" xlink:href="{___ASSET___5}"></use></g><g data-mml-node="mo" transform="translate(5089.1,0)"><use data-c="2B" xlink:href="{___ASSET___1}"></use></g><g data-mml-node="mo" transform="translate(6089.3,0)"><use data-c="28" xlink:href="{___ASSET___2}"></use></g><g data-mml-node="mi" transform="translate(6478.3,0)"><use data-c="1D465" xlink:href="{___ASSET___0}"></use></g><g data-mml-node="mo" transform="translate(7272.6,0)"><use data-c="2212" xlink:href="{___ASSET___3}"></use></g><g data-mml-node="mn" transform="translate(8272.8,0)"><use data-c="32" xlink:href="{___ASSET___6}"></use></g><g data-mml-node="mo" transform="translate(8772.8,0)"><use data-c="29" xlink:href="{___ASSET___5}"></use></g><g data-mml-node="mo" transform="translate(9384,0)"><use data-c="2B" xlink:href="{___ASSET___1}"></use></g><g data-mml-node="mo" transform="translate(10384.2,0)"><use data-c="28" xlink:href="{___ASSET___2}"></use></g><g data-mml-node="mi" transform="translate(10773.2,0)"><use data-c="1D465" xlink:href="{___ASSET___0}"></use></g><g data-mml-node="mo" transform="translate(11567.4,0)"><use data-c="2212" xlink:href="{___ASSET___3}"></use></g><g data-mml-node="mn" transform="translate(12567.7,0)"><use data-c="33" xlink:href="{___ASSET___7}"></use></g><g data-mml-node="mo" transform="translate(13067.7,0)"><use data-c="29" xlink:href="{___ASSET___5}"></use></g><g data-mml-node="mo" transform="translate(13678.9,0)"><use data-c="2B" xlink:href="{___ASSET___1}"></use></g><g data-mml-node="mo" transform="translate(14679.1,0)"><use data-c="2026" xlink:href="{___ASSET___8}"></use></g><g data-mml-node="mo" transform="translate(16073.3,0)"><use data-c="2B" xlink:href="{___ASSET___1}"></use></g><g data-mml-node="mn" transform="translate(17073.6,0)"><use data-c="31" xlink:href="{___ASSET___4}"></use></g><g data-mml-node="mo" transform="translate(17851.3,0)"><use data-c="3E" xlink:href="{___ASSET___9}"></use><use data-c="3D" xlink:href="{___ASSET___10}" transform="translate(778,0)"></use></g><g data-mml-node="mi" transform="translate(19685.1,0)"><use data-c="1D441" xlink:href="{___ASSET___11}"></use></g></g></g></svg></mjx-container></span></p>
```

**Comment by bluwy** [comment]
Thanks for the detailed explanation! I've added the markup to the test case and made a fix in 0.1.4. It should work without additional configuration now.

**Comment by kdheepak** [comment]
Thank you!
